### PR TITLE
feat: build workflow

### PR DESCRIPTION
**Pull request by avsetsin**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Compile
+        id: compile
+        run: |
+          echo ${{ matrix.platform }} | {
+            IFS='/' read -r GOOS GOARCH
+
+            export GOOS GOARCH CGO_ENABLED=0
```
Comment (sergeyWh1te) [inline .github/workflows/build.yml:25]: It switches off cross compiling for all apps.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+            go build -o build/dc4bc_d ./cmd/dc4bc_d/
+            go build -o build/dc4bc_cli ./cmd/dc4bc_cli/
+            go build -o build/dc4bc_airgapped ./cmd/airgapped/
```
Comment (sergeyWh1te) [inline .github/workflows/build.yml:29]: It makes sense here CGO_ENABLED=0

Comment (avsetsin) [reply .github/workflows/build.yml:29]: I have disabled that for all apps

File: .github/workflows/build.yml (hunk 1 above)
Comment (avsetsin) [reply .github/workflows/build.yml:25]: Yes, so that the build is reproducible not only for the airgapped app

**Pull request merged**